inference-engine/src/vpu/common/include/vpu/utils/containers.hpp

```diff
@@ -0,0 +1,40 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <vector>
+#include <algorithm>
+
+#include "error.hpp"
+
+namespace vpu {
+
+template<class Key, class Value, template<class...> class Map>
+inline std::vector<Key> getKeys(const Map<Key, Value>& map) {
+    auto keys = std::vector<Key>{};
+    keys.reserve(map.size());
+    std::transform(map.cbegin(), map.cend(), std::back_inserter(keys), [](const std::pair<Key, Value>& entry) { return entry.first; });
```

Comment:
seems that strait `for (auto& v : map) keys.push_back(v.first);` is shorter

Reply:
Looks like subjective point of view - I prefer STL algorithm usage if possible.

Reply:
@apankratovantonp 

Can this discussion be resolved?